From ae34127a40028a46c86f7d89f3a7209881a834af Mon Sep 17 00:00:00 2001
From: Myron Stowe <myron.stowe@redhat.com>
Date: Fri, 5 Sep 2014 01:31:45 -0400
Subject: [iommu] add missing include

Message-id: <20140905013145.30867.23654.stgit@gir.stowe>
Patchwork-id: 91270
O-Subject: [RHEL7.1 PATCH 06/28] iommu: add missing include
Bugzilla: 1129808
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Don Dutile <ddutile@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: Alex Williamson <alex.williamson@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1129808
Upstream Status: cbff5634dcb78ae2c3a687de2b097fd26967604d

commit cbff5634dcb78ae2c3a687de2b097fd26967604d
Author: Brian Norris <computersforpeace@gmail.com>
Date:   Wed Dec 4 17:22:53 2013 -0800

    iommu: add missing include

    Fix a warning in of_iommu.c:

    drivers/iommu/of_iommu.c:38:5: warning: no previous prototype for 'of_get_dm

    Signed-off-by: Brian Norris <computersforpeace@gmail.com>
    Cc: Hiroshi DOYU <hdoyu@nvidia.com>
    Signed-off-by: Joerg Roedel <joro@8bytes.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/iommu/of_iommu.c b/drivers/iommu/of_iommu.c
index ee249bc..e550ccb 100644
--- a/drivers/iommu/of_iommu.c
+++ b/drivers/iommu/of_iommu.c
@@ -20,6 +20,7 @@
 #include <linux/export.h>
 #include <linux/limits.h>
 #include <linux/of.h>
+#include <linux/of_iommu.h>
 
 /**
  * of_get_dma_window - Parse *dma-window property and returns 0 if found.
-- 
1.7.1